.vscode/tasks.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Looks like an accidental commit. (If it isn’t, putting it in a separate PR might be better.)